Some simple tweaks:

Spazzze opened the pull request:
- added option to separately configure animation duration for show and close;
```java
MenuParams menuParams = new MenuParams();
menuParams.setShowAnimationDuration(50);
menuParams.setHideAnimationDuration(0);
```
- added option to configure is click on text works same as click on icon or not;
```java
menuParams.setTextClickable(false);
```
- adding opportunity to set title by @StringRes resource, not only by String;
```java
MenuObject favourites = new MenuObject(R.string.header_favourites);
```
- changed option bgColor at MenuObject to @ColorRes and in utils now icon's background color sets by _setBackgroundColor(ContextCompat.getColor(R.color.id))_
```java
favourites.setBgColor(R.color.color_accent);
```
- added option to set id to menu item for alternative use in clickListener instead of position;
```java
favourites.setId(R.id.favourites_menu);
```
and
```java
    @Override
    public void onMenuItemClick(View v, int position) {
        switch (v.getId()) {
            case R.id.favourites_menu:
                //Do something here
                break;
        }
    }

```

TeeRawk commented on lib/src/main/java/com/yalantis/contextmenu/lib/ContextMenuDialogFragment.java:
```diff
@@ -49,25 +49,27 @@ public static ContextMenuDialogFragment newInstance(int actionBarSize, List<Menu
         params.setAnimationDelay(animationDelay);
-        params.setAnimationDuration(animationDuration);
+        params.setShowAnimationDuration(animationDuration);
+        params.setHideAnimationDuration(hideAnimationDuration);
```
Please also make a constructor without hideAnimationDuration, so it will set by default. Backward compatibility.

Spazzze replied:
> Please also make a constructor without hideAnimationDuration, so it will set by default. Backward compatibility.
My bad -_-"

- fixed mistake with changed deprecated constructor's params. We don't need 2 new deprecated constructors, we just need Backward compatibility.
- corrected params declarations in MenuObject.

Spazzze commented:
> Please also make a constructor without hideAnimationDuration, so it will set by default. Backward compatibility.
My bad -_-"

- fixed mistake with changed deprecated constructor's params. We don't need 2 new deprecated constructors, we just need Backward compatibility.
- corrected params declarations in MenuObject.

Spazzze commented:
Кстати ребят, я у себя также убрала зависимость от deprecated библиотеки com.nineoldandroids:library:2.4.0. Я не знаю, почему вы именно её до сих пор используете, могу ошибаться, но все необходимые классы и анимации есть в родном функционале при чем с теми же названиями. Убрав зависимость, единственное что пришлось сделать, это написать свой ViewHelper, в котором я убрала ненужную для себя поддержку версий  < 11 и все заработало прекрасно.
Если захочется посмотреть, то это вот [тут](https://github.com/Spazzze/DevIntensive/pull/10/commits/00259041131e1147b9a49afa60f61e91cd2bb5a0)
